helm/postgres/templates/postgres.yaml

```
@@ -191,7 +191,9 @@ spec:
   {{- if .Values.standby }}
   standby:
     enabled: {{ .Values.standby.enabled }}
-    repoName: {{ required "repoName must be set when enabling standby mode." .Values.standby.repoName }}
+    repoName: {{ .Values.standby.repoName }}
```

should be more flexible (with lease code):

```suggestion
{{- with .Values.standby }}
  standby:
    {{ toYaml . | nindent 4 }}
{{- end }}
```

---

More flexible, but the actual resource definition accepts only certain values anyway, and with so few values this seems to be an acceptable number of lines of code.